src/status_im/ui/screens/chat/styles/input/input.cljs

```diff
@@ -13,49 +13,66 @@
    :margin-bottom    margin-bottom
    :flex-direction   :column
    :border-top-width border-height
-   :border-top-color colors/black-transparent
-   :elevation        2})
+   :border-top-color colors/gray-lighter})
 
 (def reply-message
   {:flex-direction :row
    :align-items    :flex-start
-   :border-width   1
-   :border-radius  10
-   :border-color   colors/black-transparent
-   :padding-top    10
-   :padding-bottom 10
-   :padding-right  14
-   :padding-left   7
-   :margin-top     12
-   :margin-left    12
-   :margin-right   12})
+   :justify-content :space-between
+   :padding-top    8
+   :padding-bottom 8
+   :padding-right  8
+   :padding-left   8})
 
 (def reply-message-content
   {:flex-direction :column
-   :padding-left   7
+   :padding-left   8
+   :padding-right 8
    :max-height     140})
 
 (defn reply-message-author [chosen?]
   (assoc (message-author-name chosen?)
-         :padding-bottom 6))
+         :flex-shrink 1
+         :padding-left 0
```

Comment:
does 0 padding and 0 margin change anything?

Reply:
Yep, it's overriding the values from the definition of `message-author-name`, which is

```clojure
(defn message-author-name [chosen?]
  {:font-size           (if chosen? 13 12)
   :font-weight         (if chosen? "500" "400")
   :padding-top         6
   :padding-left        12
   :padding-right       16
   :margin-right        12
   :text-align-vertical :center
   :color               colors/gray})
```